doc/news/changes/minor/20160311TobyD

```diff
@@ -0,0 +1,3 @@
+New: Added unit tests for complex-valued PETSc and SLEPc.
+<br>
+(Toby D. Young, Denis Davydov, 2016/03/11)
```

Comment:
I think that this is the last problematic case left - I couldn't spot any more issues or instances of people using special characters in their name.